utils/bazel/llvm-project-overlay/llvm/BUILD.bazel

```diff
@@ -4178,6 +4187,12 @@ cc_binary(
     ],
 )
 
+llvm_driver_cc_binary(
+    name = "llvm-nm",
+    stamp = 0,
```

Comment:
Do we need guidance when and where we use `stamp = 0`  （https://bazel.build/reference/be/c-cpp） as a comment?

Reply:
Good question. Some people may have strong opinions on stamping. I'm not sure if there's a reason we have `stamp = 0` in these build files. Obviously there are caching benefits, but you also get those more broadly by building with `bazel build --nostamp`. I can't find advice on this, e.g. in a style guide or best practices list, but I'll add a comment later if I can figure anything out.

(btw, this looks like a new line, but it's just moved from a few lines above)